packages/plugin-page-view-v1-enrichment/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@amplitude/plugin-page-view-v1-enrichment-browser",
+  "version": "0.0.0",
```

Comment:
Are you going to bump this version later?

Reply:
Yes, it will be bumped up by the release.

https://github.com/amplitude/Amplitude-TypeScript/pull/503/files#diff-235400fe4397dc5a6273af81b761054e1729653c744418fb046a2e6e5bfcd6de